app/src/main/java/com/droidhats/campuscompass/CalendarFragment.kt

```diff
@@ -0,0 +1,30 @@
+package com.droidhats.campuscompass
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+
+class CalendarFragment : Fragment() {
+
+    private lateinit var calendarViewModel: CalendarViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        calendarViewModel =
+            ViewModelProviders.of(this).get(CalendarViewModel::class.java)
+        val root = inflater.inflate(R.layout.calendar_fragement, container, false)
```

Comment:
fragment is misspelled so we will need to change the name of the associated file